editor-support.md

```diff
@@ -131,12 +130,49 @@ inspection.
 
 ## VSCode
 
-[VSCode](https://code.visualstudio.com/) with [commonlisp-vscode extension](https://marketplace.visualstudio.com/items?itemName=ailisp.commonlisp-vscode)
-supports running a REPL, evaluate code, auto indent, code completion, go to definition, documentation on hover, etc.
-It's based on [cl-lsp](https://github.com/ailisp/cl-lsp) language server and it's possible to write LSP client that works in other editors.
+[VSCode](https://code.visualstudio.com/) now has excellent support for
+Common Lisp via a few different solutions.
+
+[Alive](https://marketplace.visualstudio.com/items?itemName=rheller.alive)
+hooks directly into the SWANK server that emacs SLIME uses.  It
+has no dependencies beyond a version of Common Lisp on which to run
+SWANK.  It currently supports:
+- Syntax highlighting
+- Code completion
+- Code formatter
+- Jump to definition
+- Snippets
+- REPL integration
```

Comment:
Does it have the interactive debugger, with explorable stacktraces?

Reply:
It does.  Should I put that in this list?

Reply:
I added it, and will add docs for how to use it.